src/Elastic.Markdown/BuildContext.cs

```diff
@@ -39,6 +39,15 @@ public string? UrlPathPrefix
 		init => _urlPathPrefix = value;
 	}
 
+	private readonly string? _staticUrlPathPrefix;
+	public string? StaticUrlPathPrefix
+	{
+		get => !string.IsNullOrWhiteSpace(_staticUrlPathPrefix)
+			? $"/{_staticUrlPathPrefix.Trim('/')}"
```

Comment:
Do we need to use `Path.DirectorySeparatorChar` here?

```suggestion
			? $"/{_staticUrlPathPrefix.Trim(Path.DirectorySeparatorChar)}"
```

Reply:
No this are always url paths with `/`